R/tax_unique.R

```diff
@@ -90,6 +96,7 @@
 #' @section Reviewer(s):
```

Comment:
It won't let me select it, but
"The function matches taxonomic names at increasingly higher taxonomic levels, so homonyms may be falsely filtered out."
isn't true anymore. Perhaps change to
"The function matches taxonomic names at face value, so homonyms may be falsely filtered out."